bijection-util/src/main/scala/com/twitter/bijection/twitter_util/UtilBijections.scala

```diff
@@ -122,6 +123,12 @@ trait UtilBijections {
       override def apply(pool: FuturePool) = new TwitterExecutionContext(pool)
       override def invert(context: ExecutionContext) = new ScalaFuturePool(context)
     }
+
+  implicit def byteArrayBufBijection: Bijection[Array[Byte], Buf] =
+    new AbstractBijection[Array[Byte], Buf] {
+      override def apply(bytes: Array[Byte]) = Buf.ByteArray.Owned(bytes)
```

Comment:
Hmm, maybe we should namespace them slightly differently and supply both?  I feel like the point of Owned vs Shared was that we wanted folks to have to think about what it was OK to do with the data, so maybe we should do it here too (since we intend for this to be an extension of our API).